_posts/2024-01-04-release-2.7.0.md

```diff
@@ -0,0 +1,125 @@
+  - **MacOS** (on [x86-64][] and [ARM64][])
+
+
+## Upgrading to MDAnalysis version 2.7.0
+
+To update with `conda` from the [conda-forge channel][] run
+
+```bash
+conda update -c conda-forge mdanalysis
+```
+
+To update from [PyPi][] with `pip` run
+
+```bash
+python -m pip install --upgrade MDAnalysis
+```
+
+For more help with installation see the [installation instructions in the User Guide][]. 
+Make sure you are using a Python version compatible with MDAnalysis 
+before upgrading (Python >= 3.9).
+
+
+## Notable changes
+
+For a full list of changes, bugfixes and deprecations see the [CHANGELOG][].
+
+
+#### Fixes:
+- NoJump now properly handles jumps that occur on the second frame of NPT
+  trajectories (PR #4258).
+- Fixed charge reading from PDBQT files (PR #4283).
+- Fixed a case where qcprot.CalcRMSDRotationalMatrix would return a RMSD
+  of None (PR #4273).
+
+#### Enhancements:
+- Support was added for reading chainID from prmtop AMBER topologies (PR #4007).
+- Added support for Python 3.12 (PR #4309, #4300, #4301, #4319, #4325,
+  #4327, #4329)
+- Added support for reading chainID from Autodock PDBQT files (PR #4284), GROMACS 
+  TPR topologies (PR #4281) and amber prmtop topologies (PR #4007).
+- Various improvements to the organization and performance of Major and Minor
+  Pair analyses (PR #3735).
+- C distance backend is now exposed via libmdanalysis.pxd (PR #4342).
+- Added a GROMOS11 Reader (PR #4294).
+
+#### Changes:
+- Added mda_xdrlib as a core dependency to replace the now deprecated Python
+  xdrlib code (PR #4271).
+- ConverterBase has been moved to MDAnalysis.converters.base (PR #4253).
+- networkx is now an optional dependency of MDAnalysis (PR #4331).
+- BioPython is now an optional dependency of MDAnalysis (PR #4332).
+- Results for WatsonCrickDist nucleic acids analysis are now stored in
+  analysis.nucleicacids.WatsonCrickDist.results.distances (PR #3735).
+
+#### Deprecations:
+- Importing ConverterBase from MDAnalysis.coordinates.base will not be possible
+  after MDAnalysis 3.0 (PR #4253).
+- Deprecation with intent of removal in MDAnalysis v3.0 of the X3DNA legacy
+  code (PR #4333).
+- Deprecation with intent of removal in MDAnalysis v3.0 of the TRZ reader and
+  writer (PR #4335).
+- Deprecation with intent of removal in MDAnalysis v3.0 of the
+  MDAnalysis.lib.util.which method (PR #4340).
+- The asel argument of the timeseries attribute of Readers is now deprecated
+  in favour of the atomgroup argument (PR #4343).
+- In nucleicacids.WatsonCrickDist, accepting lists of Residue objects was
+  deprecated in favor of using ResidueGroup: using List[Residue] will be
+  removed in release 3.0.0; instead use a ResidueGroup (PR #3735).
+- In nucleicacids.WatsonCrickDist the result results.pair_distances was
+  deprecated and will be removed in 3.0.0; use results.distances instead (PR #3735).
+
+## Author statistics
+
+This release was the work of 24 contributors, **5** of which are **new contributors**.
```

Comment:
I've got 13 contributors via the changelog?

Reply:
(I got 24 from what github says comparing the 2.7.0 and 2.6.1 tags - I haven't gone through to check but I'd guess it might be including some extra names due to some kind of github shenanigans; I'll go with the changelog list instead)